### PR TITLE
New indicators layouts

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23383

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/Layouts/layout-indicatorsDetails-Domain.json [hunk 1]
```diff
@@ -0,0 +1,708 @@
+{
+    "TypeName": "Domain",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsDetails-Domain.json:2]: make sure `typeId` and `TypeName` are the same as in the previous layout

File: Packs/CommonTypes/Layouts/layout-indicatorsDetails-Domain.json [hunk 2]
```diff
@@ -0,0 +1,708 @@
...
+    "kind": "indicatorsDetails",
+    "layout": {
+        "TypeName": "",
+        "id": "f437ab4a-d045-4c43-896b-b703c41252ca",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsDetails-Domain.json:6]: make sure `id` is the same as in the previous layout

**Comment by Shellyber** [comment]
@kirbles19 Please review the release notes.

File: Packs/CommonTypes/Layouts/layout-indicatorsDetails-IP.json [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+                        "y": 0
+                    }
+                ],
+                "type": "custom"
+            }
+        ],
+        "typeId": "d7707a02-b8e3-4014-8c05-cdf08219b913",
+        "version": -1
+    },
+    "typeId": "d7707a02-b8e3-4014-8c05-cdf08219b913",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsDetails-IP.json:535]:
```suggestion
    "typeId": "ipRep",
```

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-Domain.json [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+                        "static": false,
+                        "w": 1,
+                        "x": 0,
+                        "y": 6
+                    }
+                ],
+                "type": "custom"
+            }
+        ],
+        "typeId": "f437ab4a-d045-4c43-896b-b703c41252ca",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-Domain.json:354]:
```suggestion
        "typeId": "domainRepUnified",
```

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-Domain.json [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+        "version": -1
+    },
+    "typeId": "f437ab4a-d045-4c43-896b-b703c41252ca",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-Domain.json:357]:
```suggestion
    "typeId": "domainRepUnified",
```

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-File.json [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+                        "static": false,
+                        "w": 1,
+                        "x": 0,
+                        "y": 6
+                    }
+                ],
+                "type": "custom"
+            }
+        ],
+        "typeId": "d784a480-865d-4607-8bae-cec06af12752",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-File.json:252]:
```suggestion
        "typeId": "unifiedFileRep",
```

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-File.json [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+        "version": -1
+    },
+    "typeId": "d784a480-865d-4607-8bae-cec06af12752",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-File.json:255]:
```suggestion
    "typeId": "unifiedFileRep",
```

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-IP.json [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+                        "static": false,
+                        "w": 1,
+                        "x": 0,
+                        "y": 3
+                    }
+                ],
+                "type": "custom"
+            }
+        ],
+        "typeId": "d7707a02-b8e3-4014-8c05-cdf08219b913",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-IP.json:162]:
```suggestion
        "typeId": "ipRep",
```

File: Packs/CommonTypes/Layouts/layout-indicatorsQuickView-IP.json [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+        "version": -1
+    },
+    "typeId": "d7707a02-b8e3-4014-8c05-cdf08219b913",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsQuickView-IP.json:165]:
```suggestion
    "typeId": "ipRep",
```

**Comment by ShirleyDenkberg** [comment]
@Shellyber Done

File: Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json [hunk 10]
```diff
@@ -7,587 +7,957 @@
...
-                        "static": false,
-                        "w": 3,
-                        "x": 0,
-                        "y": 13
-                    }
-                ],
-                "type": "custom"
-            }
-        ]
+			{
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json:10]: please fix indentation

File: Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json [hunk 11]
```diff
@@ -7,587 +7,957 @@
...
+						"y": 0
+					}
+				],
+				"type": "custom"
+			}
+		]
+    },
+    "indicatorsQuickView": {
+        "tabs": [
+			{
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json:722]: please fix indentation

File: Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json [hunk 12]
```diff
@@ -7,587 +7,957 @@
...
+						"moved": false,
+						"name": "Hashes",
+						"static": false,
+						"w": 1,
+						"x": 0,
+						"y": 6
+					}
+				],
+				"type": "custom"
+			}
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json:960]: please fix indentation

**Pull request merged**